clang/unittests/AST/StructuralEquivalenceTest.cpp

```diff
@@ -2066,9 +2076,9 @@ TEST_F(StructuralEquivalenceCacheTest, TemplateParmDepth) {
   ASSERT_EQ(D1->getTemplateDepth(), 0u);
 
   StructuralEquivalenceContext Ctx_NoIgnoreTemplateParmDepth(
-      get<0>(TU)->getASTContext(), get<1>(TU)->getASTContext(),
-      NonEquivalentDecls, StructuralEquivalenceKind::Default, false, false,
-      false, false);
+      get<0>(TU)->getASTContext().getLangOpts(), get<0>(TU)->getASTContext(),
+      get<1>(TU)->getASTContext(), NonEquivalentDecls,
+      StructuralEquivalenceKind::Default, false, false, false, false);
```

Comment:
```suggestion
      StructuralEquivalenceKind::Default, /*StrictTypeSpelling=*/false, /*Complain=*/false);
```

The other two default to `false` so we can leave them out.